src/components/Display.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/destructuring-assignment */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display(props) {
+  Display.propTypes = {
+    result: PropTypes.string.isRequired,
+  };
+  return (
+    <div className="display">
+      {' '}
+      {props.result || 0}
+      {' '}
```

Comment:
with `result` being string, according to your propTypes, this value should be `"0"` instead of `0`

Also, re-inforce this default-value, with `defaultProps `